Mwe events branch

savary opened the pull request:
A few MWE section events to add

savary committed:
Adding the MWE-WN 2019 workshop

savary committed:
Minor changes to MWE-WN

savary committed:
Adding the LAW-MWE-CxG 2018 workshop

savary committed:
Updating the paper submission deadlines for MWE-WN 2019 and LAW-MWE-CxG 2018

savary committed:
Adding the MWE 2017 workshop

savary committed:
Adding the MWE 2017 workshop

savary committed:
Adding the MWE 2016 workshop

savary committed:
Adding the MWE 2014 workshop

savary committed:
Adding europhras 2019 AND mumttt 2019

savary committed:
Adding MUMTTT 2017

savary committed:
Adding MUMTTT 2015

savary committed:
Adding MUMTTT 2013

bethard commented on _posts/events/2014-04-26-mwe.md:
```diff
@@ -0,0 +1,82 @@
+
+Submission and reviewing will be electronic, managed by the START system:
+
+https://www.softconf.com/eacl2014/MWE/
```
Put `<` and `>` around the URL so that it is rendered as a link.

savary replied:
Done

bethard commented on _posts/events/2015-06-04-mwe.md:
```diff
@@ -0,0 +1,74 @@
+---
+The 11th Workshop on Multiword Expressions (MWE 2015)
+
+NAACL 2015 (Denver, Colorado, USA), June 4, 2015
```
You probably want two spaces at the end of this line so that the URL shows up on its own line.

savary replied:
Done

bethard commented on _posts/events/2015-06-04-mwe.md:
```diff
@@ -0,0 +1,74 @@
+
+Submission and reviewing will be electronic, managed by the START system:
+
+https://www.softconf.com/naacl2015/mwe
```
Put `<` and `>` around the URL so that it is rendered as a link.

savary replied:
Done

bethard commented on _posts/events/2015-07-01-mumttt.md:
```diff
@@ -0,0 +1,61 @@
+SUBMISSIONS 
+
+We invite expressions of interest by 1 November 2014 by email to both contact editors below, including the chapter title, list of prospective authors and affiliation, as well as a 250-word abstract of your contribution.
+The full submission is expected to be approximatively 5,000 words in length, excluding citations and appendices, and should be formatted according to the publisher’s author guidelines (https://benjamins.com/jbp/additional/jb-guidelines-manuscript-submission-apa.pdf). The deadline for the final contributions is 31 March 2015.
```
Put `<` and `>` around the URL so that it is rendered as a link.

savary replied:
Done

bethard commented on _posts/events/2016-08-11-mwe.md:
```diff
@@ -0,0 +1,74 @@
+
+Submission and reviewing will be electronic, managed by the START system:
+
+https://www.softconf.com/acl2016/mwe
```
Put `<` and `>` around the URL so that it is rendered as a link.

savary replied:
Done

bethard commented on _posts/events/2017-11-14-mumttt.md:
```diff
@@ -0,0 +1,74 @@
+---
+The 3rd Workshop on Multi-word Units in Machine Translation and Translation Technology (MUMTTT 2017)
+
+London, UK, 13-14 November 2017
```
You probably want two spaces at the end of this line so that the URL shows up on its own line.

savary replied:
Done

bethard commented on _posts/events/2017-11-14-mumttt.md:
```diff
@@ -0,0 +1,74 @@
+
+BEST PAPER AWARD
+
+The “Best Paper at the MUMTTT workshop” award will be granted by the EUROPHRAS’2017 Conference to the authors of outstanding work on multi-word unit translation. Details on the selection criteria are provided at http://rgcl.wlv.ac.uk/europhras2017.
```
Put `<` and `>` around the URL so that it is rendered as a link.

bethard commented on _posts/events/2017-11-14-mumttt.md:
```diff
@@ -0,0 +1,74 @@
+Submissions must consist of full-text papers (6 to 8 pages for content, plus additional pages for bibliographic references). They must be formatted according to the ACL 2017 style guidelines available both for Word and LaTeX text processor. Each submission will be reviewed by at least three programme committee members. As reviewing will be double blind, papers must not reveal authors’ identity. A ...
+
+Submission is electronic, using the START conference management software at:
+https://www.softconf.com/i/mumttt2017.
```
Put `<` and `>` around the URL so that it is rendered as a link.

bethard commented on _posts/events/2018-08-25-mwe.md:
```diff
@@ -0,0 +1,124 @@
+All papers will be presented orally or as posters, as determined by the Program Committee chairs. No distinction between papers presented orally or as posters is made in the workshop proceedings.
+
+For all types of submission, the COLING 2018 LaTeX templates should be used. All papers should be submitted via the START space:
+https://www.softconf.com/coling2018/ws-LAW-MWE-CxG-2018/
```
Put `<` and `>` around the URL so that it is rendered as a link.

bethard commented on _posts/events/2019-08-02-mwe.md:
```diff
@@ -0,0 +1,95 @@
+Joint Workshop on Multiword Expressions and WordNet (MWE-WN 2019)
+
+ACL 2019 (Florence, Italy), August 2nd, 2019  
+<http://multiword.sourceforge.net/mwewn2019>  
```
Maybe you want another blank line after the link?

savary replied:
Done

bethard commented on _posts/events/2019-08-02-mwe.md:
```diff
@@ -0,0 +1,95 @@
+
+The dissemination track submissions are not anonymous, and they should not exceed one page, including the authors' names and affiliations, the mention of the original venue, the link to the original paper and a short explanation why the paper is relevant to MWEs and Wordnets workshop. If the original paper is not publicly available, it should also be submitted in a separate .pdf file but it does  ...
+
+All papers should be submitted via the following START space: https://www.softconf.com/acl2019/mwewn
```
Put `<` and `>` around the URL so that it is rendered as a link.

savary replied:
Done

bethard commented on _posts/events/2019-08-02-mwe.md:
```diff
@@ -0,0 +1,95 @@
+ * Jelena Mitrović, University of Passau (Germany)
+ * Verginica Barbu Mititelu, Romanian Academy Research Institute for Artificial Intelligence (Romania)
+
+The organizers will also chair the program committee.  The full committee is shown on http://multiword.sourceforge.net/mwewn2019/.
```
Put `<` and `>` around the URL so that it is rendered as a link.

bethard commented on _posts/events/2019-09-25-europhras.md:
```diff
@@ -0,0 +1,62 @@
+---
+Computational and Corpus-based Phraseology (Europhras 2019)
+
+Malaga, 25-27 September 2019
```
You probably want two spaces at the end of this line so that the URL shows up on its own line.

savary replied:
Done

bethard commented on _posts/events/2019-09-25-europhras.md:
```diff
@@ -0,0 +1,62 @@
+
+The above list is indicative and not exhaustive. Any submission presenting a study related to the alternative terms of phraseological units, multiword expressions, multiword units, formulaic language, or polylexical expressions, will be considered.
+
+SUMBMISSIONS ADN PUBLICATION
```
"AND" instead of "ADN"?

savary replied:
Done

bethard commented on _posts/events/2019-09-27-mumttt.md:
```diff
@@ -0,0 +1,77 @@
+---
+The 4th Workshop on Multi-word Units in Machine Translation and Translation Technology (MUMTTT 2019)
+
+Malaga, 27 September 2019
```
You probably want two spaces at the end of this line so that the URL shows up on its own line.

savary replied:
Done

bethard commented on _posts/events/2019-09-27-mumttt.md:
```diff
@@ -0,0 +1,77 @@
+
+They must be formatted according to the ACL 2017 style guidelines available both for Word and LaTeX text processor. Each submission will be reviewed by at least three programme committee members. Accepted papers will be presented orally or as posters, as determined by the programme committee. There will be no distinction in the workshop proceedings between papers presented orally or as posters. T ...
+
+All submissions must be made online using the Softconf START conference management system:   https://www.softconf.com/j/mumttt2019.
```
Put `<` and `>` around the URL so that it is rendered as a link.

savary replied:
Done

bethard reviewed (commented):
Thanks for putting this together. I made a few small Markdown suggestions, but otherwise it looks good to me. @ceramisch, feel free to merge once those things are fixed.

bethard commented on _posts/events/2016-08-11-mwe.md:
```diff
@@ -0,0 +1,74 @@
+event_location: "Berlin, Germany"
+event_submission_date: "2016-05-08"
+host_name: "ACL 2016"
+host_url: "https://www.aclweb.org/portal/content/acl-2016-annual-meeting-association-computational-linguistics-0"
```
Maybe https://mirror.aclweb.org/acl2016/ instead?

savary replied:
Done

bethard commented on _posts/events/2018-08-25-mwe.md:
```diff
@@ -0,0 +1,124 @@
+event_url: "http://multiword.sourceforge.net/lawmwecxg2018"
+event_location: "Santa Fe, USA"
+event_submission_date: "2018-05-27"
+host_name: "COLING 20"
```
Maybe COLING 2018?

savary replied:
Done

savary committed:
Update 2014-04-26-mwe.md

savary committed:
Update 2015-06-04-mwe.md

Two spaces added before a URL

savary committed:
Update 2015-06-04-mwe.md

< and > added around a URL

savary committed:
Update 2015-07-01-mumttt.md

< and > added around a URL

savary committed:
Update 2016-08-11-mwe.md

< and > added around a URL

savary committed:
Update 2016-08-11-mwe.md

https://www.aclweb.org/portal/content/acl-2016-annual-meeting-association-computational-linguistics-0 replaced by https://mirror.aclweb.org/acl2016/

savary committed:
Update 2017-11-14-mumttt.md

Two spaced added after a line

savary committed:
Update 2018-08-25-mwe.md

COLING 20 changed to COLING 18

savary committed:
Update 2019-08-02-mwe.md

A newline added

savary committed:
Update 2019-09-25-europhras.md

Two spaces added after a line

savary committed:
Update 2019-09-27-mumttt.md

Two spaces added after a line

savary reviewed (commented):
All suggested changes applied

savary committed:
Update 2019-09-27-mumttt.md

< and > added around a URL

savary committed:
Update 2019-09-25-europhras.md

ADN canged to AND

savary committed:
Update 2019-08-02-mwe.md

< and > added around a URL

savary commented:
In fact 3 were missing. Now all are done.
